inc/customizer.php

```diff
@@ -233,6 +233,53 @@ public static function parse_css_rules( array $rules ) {
 
 	}
 
+	/**
+	 * Compact CSS output.
+	 *
+	 * @link  https://github.com/GaryJones/Simple-PHP-CSS-Minification
+	 * @since NEXT
+	 *
+	 * @param  string $css
+	 *
+	 * @return string
+	 */
+	public static function compact_css( $css ) {
```

Comment:
@jonathanbardo I made things better with https://github.com/godaddy/wp-primer-theme/pull/150/commits/e3fba90e80d73ab16e792da321f42e9afd2c8c8c so now media queries are properly supported. But yeah I get it, I wanted to take a stab at it - and it does work. I ran Primer's `style.css` through it and tested the output with the W3C CSS validator and got back identical results for both.